### PR TITLE
Tidied up examples in 'Automatic coin selection' in coin-selection.md

### PR DESCRIPTION
**Pull request by Bosch-0**
The list structure currently used looks messy and didn't integrate into this page as cleanly as it could. Added a clear title, a h4 to the algorithm headings. Added some spacing to the md file as well to make it easier to parse.

### CODE REVIEW

File: guide/payments/coin-selection.md [hunk 1]
```diff
@@ -128,30 +128,40 @@ A user chooses to send a payment to one of their contacts. They enter the amount
 
 </div>
 
-A few *popular algorithms* currently implemented by Bitcoin wallets:
+### Examples
 
-- **First In First Out (FIFO) / Last In, First Out (LIFO)**<br/>
+Below are some popular algorithms currently implemented by Bitcoin wallets that optimize for speed and lower fees:
+
+#### First In First Out (FIFO) / Last In, First Out (LIFO)<br/>
```
Comment (johnsBeharry) [inline guide/payments/coin-selection.md:135]: General feedback, the `<br>` makes the hash drop to the next line.

Screenshot 2021-08-06 at 13 11 12

it looks fine without it.

Screenshot 2021-08-06 at 13 11 55

**Review by johnsBeharry — CHANGES_REQUESTED** [review]

File: guide/payments/coin-selection.md (hunk 1 above)
Comment (Bosch-0) [reply guide/payments/coin-selection.md]: Fixed

File: guide/payments/coin-selection.md [hunk 2]
```diff
@@ -128,30 +128,40 @@ A user chooses to send a payment to one of their contacts. They enter the amount
...
-- **Minimize Future Fees (Merge Coins)**<br/>
+
+#### Minimize Future Fees (Merge Coins)
 Spending the maximum number of inputs to merge coins as a single change output for future use. This strategy can optimise for speed as the transaction size (and therefore cost) is increased. However, merging coins can also lead to a loss of privacy as coins, their addresses, balances, and historical transaction data are intertwined.
 
-A few more, *optimising for privacy*
+Below are algoorithms that optimize for privacy
 
-- **Target Sized Change**<br/>
+#### Target Sized Change<br/>
```
Comment (pavlenex) [inline guide/payments/coin-selection.md:152]:
```suggestion
#### Target Sized Change
```

**Review by pavlenex — APPROVED** [review]

**Comment by GBKS** [comment]
I prefer the original list look. The changed titles look too much like headers. YMMV.

**Comment by Bosch-0** [comment]
> The changed titles look too much like headers.

That's what I was going for, I think these points have enough significance to be headers.

**Comment by pavlenex** [comment]
@johnsBeharry can you review please, your `request a change` is blocking me from being able to merge the PR.

**Review by johnsBeharry — APPROVED** [review]
LGTM

**Pull request merged**